src/day8/re_frame/trace.cljs

```diff
@@ -144,20 +144,27 @@
     (fn [trace]
       (< (:query query) (:duration trace)))))
 
-(defn render-traces [showing-traces filter-items trace-detail-expansions]
+(defn add-filter [filter-items filter-input filter-type from-mouse?]
+  (swap! filter-items conj {:id (random-uuid)
+                            :query (if from-mouse?
```

Comment:
I think we can remove the `from-mouse?` parameter for now. That way we will always have the same behaviour for a certain filter type. We can then implement the exact keyword matching stuff as a new filter type (`:op-type`?) in https://github.com/Day8/re-frame-trace/issues/39.